README.md

```diff
@@ -1,21 +1,77 @@
 # golf-course
 A list of common Solidity optimization tips and myths.
 
-## Tips
+# Tips
 
-### Right Shift Instead of Dividing By 2
+- - - -
+### 1.  When dividing by two, use `>> 1` instead of `/ 2` ###
+
+```solidity
+/// 🤦 Unoptimized (gas: 1401)
+uint256 two = 4 / 2;
+
+/// 🚀 Optimized (gas: 1372)
+uint256 two = 4 >> 1;
+```
 
 The `SHR` opcode is 3 gas cheaper than `DIV` and also bypasses Solidity's division by 0 prevention overhead.
+  - [Gas Usage]()
+  - [Full Example](https://github.com/Rari-Capital/golf-course/blob/fc1882bacfec50787d9e9435d59fed4a9091fb21/src/GolfCourse.t.sol#L15)
```

Comment:
I think you mentioned we should link to the test, which is what I've done here, but I was also thinking maybe it would be good to link to the example code?

Reply:
yea i think we should do that instead of test tbh wdyt

Reply:
we could even break convention and put the example code in the test file...